_posts/2019/2019-05-21-making-a-move-abroad-from-bolivia-to-hamburg.md

```diff
@@ -0,0 +1,58 @@
+date: 2019-05-21
+header_image: public/relocation.jpg
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["onboarding", "relocation", "new job"]
+authors: ["Kathia"]
+about_authors: ["ksalazar"]
+---
+
+Hi, my name is Kathia. 
+I come from Bolivia, and since March 2019 I'm working as a Software Developer at ePages in Hamburg.
+In late 2018 I applied online, and then had a first interview with Serena, the HR recruiter.
+After the interview, I had to solve a coding challenge.
+The next week, I talked with Unai, one of the developers of my current team, to discuss a few things about my code as well as my previous experience.
+This was followed by an interview with Harm, the CTO, and Serena.
+Finally, I received the offer from ePages 10 days before Christmas.
+So, this was like an early Christmas present for me 🙂.
+
+## Why the decision was easy
+
+I learnt German in Bolivia, and reached a pretty decent level - B1 😉. 
+In 2018, I had the chance to take a one-month course in Düsseldorf, and then I spent one more month in Aachen.
+Overall this first experience strengthened my desire of living and working in Germany.
+Of course leaving friends and family is not easy. 
+Even worse, if you have an adorable dog that has been with you during the last 15 years 😔.
+But life is about challenges, so I didn't hesitate to take this opportunity to live and work in Hamburg, which is one of the most beautiful cities.
+
+## Getting the visa
+
+Since I come from a non-EU country, I had to apply for a visa to get a EU Blue Card, which is a work and residence permit that I didn't know about until the company told me that I could apply for it.
+Getting the paperwork done for the visa was easier than I expected.
+During this process, I got support from Serena, and Elisa, the HR Manager.
+My visa was approved one week after the interview at the German Embassy.
+
+## The company
+
+I had a nice welcome on my first day at work.
+That week, we had the ePages Hackathon.
+Thanks to this event, it was possible to meet the colleagues from Jena and Barcelona. 
+During the first weeks, there were a couple of meetings organized to give the new employees an overview about the role of the team at ePages.
+This way, we understand what everyone does, what technologies they use, and what projects they are working on. 
+
+## My team
+
+I'm part of the Team 42, along with Unai and German, the developers from Spain. 
+We mainly work with Ruby, Ruby on Rails, and also JavaScript.
+We have the daily meetings at 09:15 with Timo, our Product Owner, and Elisa (another Elisa!), our Scrum Master.
+
+## Getting the paperwork done
+
+Parallel to work, I had to take care of some German documents needed, such as the address registration, bank account, and health insurance.
+All that went fine.
+Once I got everything ready, I requested the Blue Card at the immigration office. 
+The only thing I can complain about is that I had to go very early... at 7 a.m!
```

Comment:
Would love to read some final impressions on how you like it so far, the decision, a little summary, next steps.